Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -3433,6 +3441,156 @@ def get_power_query_results(client: Client, args: dict):
+        accountIds=args.get('account_ids'),
+        applicationCatalogId=args.get('application_catalog_id'),
+        creator__contains=args.get('creator_contains'),
+        cursor=args.get('cursor'),
+        id=args.get('id'),
+        limit=int(args.get('limit', 1000)),
+        name__contains=args.get('name_contains'),
+        siteIds=args.get('site_ids')
+    )
+
+    # Make request and get raw response
+    installed_applications_page, pagination = client.list_installed_applications_request(query_params)
+    installed_applications.extend(installed_applications_page)
+
+    while pagination and pagination.get("nextCursor"):
+        demisto.debug("Got the next page for installed applications \n {}".format(pagination['nextCursor']))
+        query_params['cursor'] = pagination['nextCursor']
+        if query_params.get('limit'):
+            del query_params['limit']
+        installed_applications_page, pagination = client.list_installed_applications_request(query_params)
+        installed_applications.extend(installed_applications_page)
+
+    all_scopes = []
+    if installed_applications:
+        for each_app in installed_applications:
+            scopes = each_app.get("scopes")
+            if scopes is not None and len(scopes) > 0:
+                for scope in scopes:
+                    scope["applicationCatalogId"] = each_app["applicationCatalogId"]
+                    scope["applicationCatalogName"] = each_app["name"]
+                    all_scopes.append(scope)
+        meta = "Provides summary information and details for all the installed applications that matched your search criteria."
+    else:
+        meta = "The search filters provided are returning no results. Please review and adjust them accordingly."
+
+    context_entries = []
+    for each_scope in all_scopes:
+        entry = {
+            'ID': each_scope.get('id'),
+            'Account': each_scope.get('account'),
+            'AccountId': each_scope.get('accountId'),
+            'ApplicationCatalogId': each_scope.get('applicationCatalogId'),
+            'applicationCatalogName': each_scope.get('applicationCatalogName'),
+            'AlertMessage': each_scope.get('alertMessage'),
+            'CreatedAt': each_scope.get('createdAt'),
+            'Creator': each_scope.get('creator'),
+            'CreatorId': each_scope.get('creatorId'),
+            'DesiredStatus': each_scope.get('desiredStatus'),
+            'HasAlert': each_scope.get('hasAlert'),
+            'LastEntityCreatedAt': each_scope.get('lastEntityCreatedAt'),
+            'Modifier': each_scope.get('modifier'),
+            'ModifierId': each_scope.get('modifierId'),
+            'ScopeId': each_scope.get('scopeId'),
+            'ScopeLevel': each_scope.get('scopeLevel'),
+            'Status': each_scope.get('status'),
+            'UpdatedAt': each_scope.get('updatedAt'),
+            'ApplicationInstanceName': each_scope.get('applicationInstanceName'),
```

Comment:
```suggestion
            'applicationInstanceName': each_scope.get('applicationInstanceName'),
```
lets make it camelCase - or make it consistent with other command's context

Reply:
do it for all